brave/brave-core
# Navigator.getBattery should return constant result

fix https://github.com/brave/brave-browser/issues/1398

## Submitter Checklist:


## Test Plan:

1. Open Brave and navigate to the Developer Console 
2. Enter: `navigator.getBattery()`

```
>navigator.getBattery()
Promise {
<resolved>: BatteryManager}
__proto__: Promise[[PromiseStatus]]: "resolved"
[[PromiseValue]]: BatteryManager
  charging: true
  chargingTime: 0
  dischargingTime: Infinity
  level: 1
  onchargingchange: null
  onchargingtimechange: null
  ondischargingtimechange: null
  onlevelchange: null
  __proto__: BatteryManager
```

3. Disconnect the charger from your device and let the battery drain a bit. Invoke `getBattery` again and verify the values remain the same.

## Reviewer Checklist:
